### PR TITLE
Style

### PR DESCRIPTION
**Pull request by nevisende**
- At this point I have a fully working app that uses only basic styles to make the layout work, according to this wireframe:
<p align="center" dir="auto">
  <a target="_blank" rel="noopener noreferrer" href="/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png">Basic UI</a>
</p>

- I improved the look and feel of the application, adding the styles of my choice.
- I keep the general layout of the wireframe
- I used pure CSS

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -1 +1 @@
-<!doctype html><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Webpack Exercise</title><script defer="defer" src="index.bundle.js"></script></head><body><h1>Hello webpack!</h1></body></html>
```
Comment (helman101) [inline dist/index.html:1]: ### I will use this section to highlight the required changes on your README file
- [ ] Your `Acknowledgments` section did not change, please use this section to add some acknowledgments or in the case you don't need it, please remove it 😄 
- [ ] The link to your `MIT` license does not take anywhere, if you are not using a license please remove this section 😄

**Review by helman101 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🔴 

Hi @nevisende 👋, 

Good job so far with this project. 👏 👏👏
You've done well implementing some of the requirements, but there are still some which aren't addressed yet.

## Required Changes ♻️

- [x] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html (hunk 1 above)
Comment (nevisende) [reply dist/index.html:1]: Okay but this is not a requirement for this project section.

Comment (nevisende) [reply dist/index.html:1]: Okay thanks :)

**Pull request merged**